jwst/tso_photometry/tests/test_tso_photometry_step.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from astropy.io import fits
+
+from jwst.tso_photometry.tso_photometry_step import TSOPhotometryStep
```

Comment:
Should this be a relative import?

Reply:
This `tests` subdirectory wasn't set up with `__init__.py`, so I left it as is. I've added the missing `__init__.py` so that relative imports can be used.